api-server/src/events/events.controller.ts

```diff
@@ -29,17 +31,54 @@ export class EventsController {
         private readonly eventsService: EventsService,
         private readonly slackNotificationService: SlackNotificationService,
         @Inject(WINSTON_MODULE_NEST_PROVIDER) private readonly logger: LoggerService
-    ) {}
+    ) { }
 
+    @Get('current-verified')
+    @ApiOperation({ summary: 'Get current events that have been verified (for public consumptions)' })
+    @ApiResponse({
+        status: 200,
+        description: 'current verified events',
+        type: EventsResponse
+    })
+    getAllCurrentVerified(
+        @Query('embed') embed: string[] | string = [],
+        @Query('tags') tags: string[] | string = []
+    ): Promise<EventsResponse> {
+        if (typeof embed === 'string') {
+            embed = [embed, 'DATE_TIME']
+        } else {
+            embed.push('DATE_TIME')
+        }
+
+        const findOptions = {
+            ...getOptionsForEventsServiceFromEmbedsQueryParam(embed),
+            where: {
```

Comment:
nice, thinks for getting this smoothed out.

Do you think this is best left here or shifted to a service method like getCurrentEvents?

Reply:
yeah i agree there's room for improvement here in future refactors 👍🏼